doc.go

```diff
@@ -21,86 +21,64 @@ Usage:
-      --regtest=           Use the regression test network
-      --nocheckpoints=     Disable built-in checkpoints.  Don't do this unless
-                           you know what you're doing.
-      --dbtype=            Database backend to use for the Block Chain (leveldb)
-      --profile=           Enable HTTP profiling on given port -- NOTE port must
-                           be between 1024 and 65536 (6060)
-      --cpuprofile=        Write CPU profile to the specified file
-  -d, --debuglevel:        Logging level for all subsystems {trace, debug, info,
-                           warn, error, critical} -- You may also specify
-                           <subsystem>=<level>,<subsystem2>=<level>,... to set
-                           the log level for individual subsystems -- Use show
-                           to list available subsystems (info)
-      --upnp               Use UPnP to map our listening port outside of NAT
-      --limitfreerelay=    Limit relay of transactions with no transaction fee
-                           to the given amount in thousands of bytes per minute
-                           (15)
-      --norelaypriority    Do not require free or low-fee transactions to have
-                           high priority for relaying
-      --maxorphantx=       Max number of orphan transactions to keep in memory
-                           (1000)
-      --generate=          Generate (mine) bitcoins using the CPU
-      --miningaddr=        Add the specified payment address to the list of
-                           addresses to use for generated blocks -- At least
-                           one address is required if the generate option is set
-      --blockminsize=      Mininum block size in bytes to be used when creating
-                           a block
-      --blockmaxsize=      Maximum block size in bytes to be used when creating
-                           a block (750000)
-      --blockprioritysize= Size in bytes for high-priority/low-fee transactions
-                           when creating a block (50000)
-      --getworkkey=        DEPRECATED -- Use the --miningaddr option instead
-      --addrindex=         Build and maintain a full address index. Currently
-                           only supported by leveldb.
-      --dropaddrindex=     Deletes the address-based transaction index from the
-                           database on start up, and the exits.
+  -V, --version             Display version information and exit
+  -C, --configfile=         Path to configuration file
+  -b, --datadir=            Directory to store data
+      --logdir=             Directory to log output.
+  -a, --addpeer=            Add a peer to connect with at startup
+      --connect=            Connect only to the specified peers at startup
+      --nolisten            Disable listening for incoming connections -- NOTE: Listening is automatically disabled if the --connect or --proxy options are
```

Comment:
This and several others no longer match the actual wrapping done by the server when `-h` is specified and it will not show up right on godoc either.